### ISSUE TITLE
Many thanks !

### ISSUE THREAD

**Issue by covid-19-stats** [victorvicpal/COVID19_es#3]
Thank you so much for keeping this git updated !
You can see data i recover from your git in use [here](https://covid-19-stats.info/?c=Spain) (no ads)

**Comment by victorvicpal** [comment]
Nice visualizations!
Thank you for using it :)

**Comment by pacovi** [comment]
Same here, thanks a lot @victorvicpal and @covid-19-stats !!

Other visualizations here:
https://datastudio.google.com/reporting/6c542aa8-04cd-400d-957f-a7120ef99caa/page/7EnIB
(no ads or whatsoever)

**Comment by covid-19-stats** [comment]
i see data is available.
https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/Actualizacion_56_COVID-19.pdf
Do you have an automated PDF to CSV programmed to recover it?

**Comment by victorvicpal** [comment]
No, I'm executing a script just after work or during my break to recover the daily data.
I will update it asap

Also, I will try to do it automatically this weekend.
I've got some problems because they change the source data every now and then.

**Comment by victorvicpal** [comment]
No, I'm executing a script just after work or during my break to recover the daily data.
I will update it asap

Also, I will try to do it automatically this weekend.
I've got some problems because they change the source data every now and then.
